### PR TITLE
Shuttle movement immediately updates lighting objects

### LINKED ISSUE #25857: Shuttle movement should immediately update lighting objects
Having that 2 seconds of darkness is really immulsion breaking

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #25857

### CODE REVIEW

File: code/modules/shuttle/on_move.dm [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/turf/proc/onShuttleMove(turf/oldturf)
```
Comment (Jalleo) [inline code/modules/shuttle/on_move.dm]: at least it doesnt move.

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -503,6 +503,8 @@
 			//move mobile to new location
 			for(var/atom/movable/AM in T0)
 				AM.onShuttleMove(T1, rotation)
+			
+			T1.onShuttleMove(T0)
```
Comment (coiax) [inline code/modules/shuttle/shuttle.dm]: what is this for

Comment (Cyberboss) [inline code/modules/shuttle/shuttle.dm]: /atom/movable/proc/onShuttleMove and /turf/proc/onShuttleMove aren't the same function

Comment (coiax) [inline code/modules/shuttle/shuttle.dm]: oh

Comment (Jalleo) [inline code/modules/shuttle/shuttle.dm]: Any chance we can get a generic helper proc?

**Pull request merged**